packages/redux-query/src/lib/requestsForTestData.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import type { ResponseBody, RequestBody } from '../types';
+import type { HttpMethod } from '../constants/http-methods';
+import { getCookie } from './cookie';
+
+type Requests = {
+  rest: {
+    [url: string]: {
+      method: string,
+      body: ResponseBody,
+      response: any,
+    },
+  },
+  gql: {
+    [query: string]: {
+      response: any,
+    },
+  },
+};
+
+const createBaseRequests = () => {
+  return {
+    rest: {},
+    gql: {},
+  };
+};
+
+const cookieName = 'amp_store_requests_for_test_data';
```

Comment:
given this is a public package on NPM, do you think there is value in adding this in a configurable way? perhaps more broadly scoped cookie name at the very least?

Reply:
Yeah, I'll change the 'amp' prefix!